themes/vscode-light-plus.json

```diff
@@ -52,7 +52,8 @@
         "panel.indent_guide_hover": "#e4e6f1",
         // Scrollbar
```

Comment:
We can also add a changelog entry in the *upcoming* section of `CHANGELOG.md`.

Something along the lines of:

```
Add scrollbar hover color. Contributed by @iamDecode, https://github.com/MadLittleMods/zed-theme-vscode-light-plus/pull/2
```

Reply:
Ah yea nice, I've added this :)